app/api/odm/model.ts

```diff
@@ -53,7 +44,7 @@ class UpdateLogHelper {
     await updatelogsModel.updateMany(
       { mongoId: { $in: affectedIds }, namespace: this.collectionName },
       { $set: { timestamp: Date.now(), deleted } },
-      { upsert: true, lean: true }
+      { lean: true }
```

Comment:
This indeed fixes the problem.

However, this opens the question as to whether there is a chance that the affectedIds array was, indeed, populated with IDs that did not exist yet on the database.  That should trigger a 'create' (or upsert) with individual IDs of the non-found values.  Still, this is required to fix the problem.

We should keep an eye on whether we have reports of syncs not updating as expected.